README.rst

```diff
@@ -88,16 +88,22 @@ Define tests.
     pg.before_test('test_example_3', {engine = 'vinyl'}, function() ... end)
 
 
-Run them.
+Run tests.
 
 .. code-block:: bash
 
     luatest --help                        # list available options
-    luatest                               # run all in ./test directory
-    luatest test/feature_test.lua         # run test by file
-    luatest test/integration              # run all within directory
+    luatest                               # run all from ./test directory
+    luatest test/integration              # run all from specified directory
+    luatest test/feature_test.lua         # run all from specified file
     luatest feature other.test_example_2  # run by group or test name
```

Comment:
I would split this example into two: now it is a bit hard to understand.

What is also not clear at the first glance: we have two hierarchies in fact:

- `test/<dir>/<...>/<test file name>_test.lua` (so one can run a directory or a file)
- `<group name>.<test case name>` (so one can run a group or a test case)

A group name is assigned by default according to a file name, but it may be arbitrary (and several groups may be in one file).

Reply:
Fixed.

Reply:
Nice, thanks!